utils/patches/computeAngle/include/cart.h

```diff
@@ -0,0 +1,164 @@
+/*!
+ * Copyright (C) 2022 Carry Assistant Trolley (C.A.T) Technologies
+ *  
+ *
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02111, USA.
+ */
+
+#ifndef CART_H
+#define CART_H
+
+#include <iostream>
+#include <tuple>
+
+#define ROC_ANGLE 0.685557
+#define START_TIME 10 //ms
+
+/**
+ * Cart class to controls general movements of the robot cart
+ */
+class Cart
+{
+public:
+        /**
+        * Callback interface to compute the angle between
+        * the RSSI values of a static beacon A
+        * and the beacon B.
+        */
+        class CompAngleCallback
+        {
+        public:
+               /**
+                * Member function called to compute the angle
+                * as and when the RSSI value changes.
+                */ 
+                virtual void computeAngle(Cart &) = 0;
+        };
+
+public:
+        /**
+         * Establishes the communication with the robot cart.
+         * 
+         * @param _defaultStartTime = START_TIME
```

Comment:
@AzvenusK Can you check if these new functions appear in regenerated Doxygen with this commenting style ? I hope a lot doesnt need to edited for that adwait